lib/mixpanel/index.js

```diff
@@ -134,7 +134,6 @@ Mixpanel.prototype.identify = function(identify){
 
   // traits
   var traits = identify.traits(traitAliases);
-  if (traits.$created) del(traits, 'createdAt');
```

Comment:
I'm not entirely sure what this line was doing here, we'll automatically transform `createdAt` into `$created`, so a bit redundant, no? cc @yields